src/loaders/assetLoader.js

```diff
@@ -132,7 +141,7 @@ module.exports = async function assetLoader() {
     }),
   );
 
-  let publicPath = `__webpack_public_path__ + ${JSON.stringify(path.join('/assets', url))}`;
+  let publicPath = `__webpack_public_path__ + ${JSON.stringify(path.join(assets, url))}`;
```

Comment:
We will probably need to add a `/` before `JSON.stringify(path.join(assets, url))`